src/Templates/ProjectTemplates/CSharp/.NETCore/ConsoleApplication/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace $safeprojectname$
+{
+    internal class Program
+    {
+        internal static void Main(string[] args)
+        {
+        }
```

Comment:
The following add conceptual baggage, IMO. Do we need them in the template?

- namespace
- visibility